dataprofiler/profilers/profiler_options.py

```diff
@@ -150,7 +175,10 @@ def __eq__(self, other: object) -> bool:
         return self.__dict__ == other.__dict__
 
 
-class BooleanOption(BaseOption):
+BooleanOptionT = TypeVar("BooleanOptionT", bound="BooleanOption")
```

Comment:
small change of just moving this to the top of the file and keep all these TypeVars in the same spot for defining